backend/canisters/proposals_bot/impl/src/updates/add_governance_canister.rs

```diff
@@ -66,18 +77,61 @@ async fn create_group(args: &Args, is_nns: bool) -> Result<ChatId, Response> {
         events_ttl: None,
         gate: None,
     };
+
     match group_index_canister_c2c_client::c2c_create_group(group_index_canister_id, &create_group_args).await {
-        Ok(group_index_canister::c2c_create_group::Response::Success(result)) => Ok(result.chat_id),
+        Ok(group_index_canister::c2c_create_group::Response::Success(result)) => Ok(MultiUserChat::Group(result.chat_id)),
         Ok(response) => Err(InternalError(format!("Unable to create group: {response:?}"))),
         Err(error) => Err(InternalError(format!(
             "Error calling 'c2c_create_group' on group_index: {error:?}",
         ))),
     }
 }
 
-fn default_description(name: &str) -> String {
+async fn create_channel(args: &Args, is_nns: bool) -> Result<MultiUserChat, Response> {
```

Comment:
It will be need to join the community first

Reply:
oh yes